nucypher/config/base.py

```diff
@@ -188,17 +188,17 @@ def generate_filename(cls, modifier: str = None) -> str:
         return filename
 
     @classmethod
-    def default_filepath(cls, config_root: str = None) -> str:
+    def default_filepath(cls, config_root: Path = None) -> Path:
         """
         Generates the default configuration filepath for the class.
 
         :return: The generated filepath string
         """
         filename = cls.generate_filename()
-        default_path = os.path.join(config_root or cls.DEFAULT_CONFIG_ROOT, filename)
+        default_path = Path(config_root or cls.DEFAULT_CONFIG_ROOT) / filename
```

Comment:
DEFAULT_CONFIG_ROOT is a Path object already can this be simplified to the following?
```suggestion
        default_path = (config_root or cls.DEFAULT_CONFIG_ROOT) / filename
```